### DISCUSSION TITLE
Language Namespace scrolls up

### DISCUSSION THREAD

**Discussion by crispyaus** [requarks/wiki#3516] [Error / Bug Report]
**Describe the bug**
When selecting a language namespace to be active it will scroll back to the drop of the name space list. Very annoying if you're trying to add a lot of language namespaces! 
**To Reproduce**
1. Go to wiki site /a/locale
2. Click on the namespace dropdown
3. Scroll down and select any language
4. It Scrolls up!

**Expected behavior**
once you select a language it should keep your current position and not scroll back up to the top o f the list!

**Host Info (please complete the following information):**
 - OS: Centos 7
 - Wiki.js version: Latest
 - Database engine: SQLite

**Comment by Smankusors** [comment]
what `Wiki.js version: Latest` you mean? 2.3.81?

**Comment by crispyaus** [comment]
Yes
